lib/sinon/util/core/every.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+// This is an `every` implementation that works for all iterables
+module.exports = function (obj, fn) {
```

Comment:
Yup, I'm doing this right now, I'm a bit of an anxious guy so I have this bad habit of pushing commits as soon as I do any rebase.